### PR TITLE
Sematext Monitoring updates

### PR DESCRIPTION
**Pull request by cpipilas**
### Monitoring updates:
- Overview
- Quick Start
- Infrastructure Monitoring
    - Overview
    - Servers
    - Containers
    - Kubernetes (new)
    - Inventory
    - Processes
- Service Monitoring
    - Overview
- Settings
- Reports & Components
- Tags
- Changelog
- FAQ

### Integrations
- Kubernetes

### Agent updates
- Container metrics

### CODE REVIEW

File: docs/monitoring/index.md [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
 
-Get started in minutes with **out-of-the-box dashboards**, [**alerts**](https://sematext.com/alerts/) and [**anomaly detection**](https://sematext.com/alerts/) rules, and ability to **analyze metrics** by a number of context-aware filters.
+Get started in minutes with **out-of-the-box dashboards** that include prebuilt charts and other numeric components for the most notable metrics. Get notified with automatic [**alerts**](https://sematext.com/alerts/) powered by [**anomaly detection**](https://sematext.com/alerts/) rules, and ability to **analyze metrics** by a number of context-aware filters.
+
+## Application and Service Monitoring
+No matter how complex your infrastructure is, you can benefit from our service autodiscovery functionality and get all the important metrics from your applications and services easily and quickly. Simply choose one of our monitoring integrations and monitor performance and health metrics for all your applications.
```
Comment (otisg) [inline docs/monitoring/index.md:9]: I'd drop the "our" from docs.

File: docs/monitoring/infrastructure.md [hunk 2]
```diff
@@ -1,35 +1,59 @@
...
+Infra Monitoring is enabled by default as soon as you install Sematext Agent.
+
 If you've set up Sematext Monitoring at some point in the past, you can easily enable Infra Monitoring by:
 
-- [Upgrading](./spm-faq/#agent-updating) your `sematext-agent` to the latest version
+- [Upgrading](./spm-faq/#agent-updating) your Sematext Agent to the [latest version](https://sematext.com/docs/agents/sematext-agent/releasenotes/)
 - Setting up Infra App token on each of your machines by running the `setup-infra` command
 
-> **Note**: For Docker monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker Monitoring in Sematext Cloud.
+> **Note**: For Docker monitoring, we suggest uninstalling the existing deprecated Agent and going through the new setup steps for Docker monitoring in Sematext Cloud.
```
Comment (otisg) [inline docs/monitoring/infrastructure.md:56]: Maybe we can remove this statement?  We added it a long time ago when we had some old agent version that we deprecated (a few years back).

File: docs/monitoring/reports-and-components.md [hunk 3]
```diff
@@ -1,62 +1,68 @@
...
 
-Combined with the **Metric Reports**, and with the help of a **charting hairline**, zoom and chart navigation tools, it is easy to see how using our **Monitoring and Logging tools in a single page** is a must-have for any DevOps team! Learn more about **[Split Screen](/guide/split-screen)**.
+Similar customized group menus can be created using [`Group By` report variables](https://sematext.com/docs/dashboards/report-variables/#groups).
 
-The set of icons in the logs search and report menu let you:
+### Add Alert
 
-- **Save as View** - Filtering and grouping data is a central part of the Sematext Cloud user interface. With Saved Views you can save your filters and groups for faster access later. If you find yourself often applying the same filters on your data then this feature can be a time saver.
-- **Add Alert** - Every query can be use as an **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data..
+Every query can be use as an **alert query**. The ability to save alerts is part of the essential tools for searching, correlating and investigating logs data..
```
Comment (otisg) [inline docs/monitoring/reports-and-components.md:27]: Extra dot at the end?

File: docs/monitoring/reports-and-components.md [hunk 4]
```diff
@@ -1,62 +1,68 @@
...
 Each alert query is visible in the Alert Rules list.
 
-- **Scheduled Reports** - There are E-mail and Subscription types available with time, frequency, and time range defined so your team can get automatated reports about your logs and system health. Learn more about **[Scheduled Reports](/guide/scheduled-reports)**.
+### Scheduled Reports
 
-- **Full Screen Mode** - Entering full screen mode brings only the component you choose into view. It's useful for presentations and viewing detailed info of a single component. All charts have zoom, crosshair, and time navigation tools with tooltips available for effective chart interaction and data analysis.
+There are E-mail and Subscription types available with time, frequency, and time range defined so your team can get automatated reports about your logs and system health. Learn more about **[Scheduled Reports](/guide/scheduled-reports)**.
```
Comment (otisg) [inline docs/monitoring/reports-and-components.md:32]: Subscription types?  Is this referring to Scheduled Reports? (we used to call them Subscriptions)

File: docs/monitoring/settings.md [hunk 5]
```diff
@@ -1,21 +1,20 @@
...
-  - heartbeat alert creation
-  - App ownership transfer
-  - scheduled report emails (aka Subscriptions)
-  
-<!-- <img class="content-modal-image" alt="Sematext Monitoring App Settings UI screen" src="../images/monitoring/app-settings-menu.png" title="Sematext Monitoring App Settings Screen"> -->
+  - [changing your App's plan](https://sematext.com/pricing/#infrastructure)
+  - [connecting Apps](https://sematext.com/docs/guide/connected-apps/)
+  - [heartbeat alert creation](https://sematext.com/docs/alerts/creating-heartbeat-alerts/)
+  - [App ownership transfer](https://sematext.com/docs/team/transfer-apps/)
+  - [scheduled report emails (aka Subscriptions)](https://sematext.com/docs/guide/scheduled-reports/)
```
Comment (otisg) [inline docs/monitoring/settings.md:14]: zap (aka Subscriptions) -- that's an ancient name

**Review by otisg — APPROVED** [review]
Left several comments.  Minor fixes needed. LGTM!

File: docs/monitoring/reports-and-components.md (hunk 4 above)
Comment (cpipilas) [reply docs/monitoring/reports-and-components.md]: Nice name btw :)

File: docs/monitoring/infrastructure.md (hunk 2 above)
Comment (cpipilas) [reply docs/monitoring/infrastructure.md]: Yes

**Pull request merged**